pitop/robotics/navigation/navigation_controller.py

```diff
@@ -276,7 +323,7 @@ def __sub_goal_reached(self):
 
     def __get_new_pose_update(self):
         self._position_update_event.wait()
-        return self.robot_state.pose
+        return self.robot_state.x, self.robot_state.y, self.robot_state.theta
```

Comment:
Do we need to return 3 parameters? Or can we group them somehow?

Reply:
`pose` grouping is what makes the most sense, but I've since changed this because of the other comment above